app/main/views/notifications.py

```diff
@@ -59,40 +46,20 @@ def view_notification(service_id, notification_id):
 
     personalisation = get_all_personalisation_from_notification(notification)
     error_message = None
-    if notification['template']['is_precompiled_letter']:
-        try:
-            file_contents, metadata = get_letter_file_data(
-                service_id, notification_id, "pdf", with_metadata=True
-            )
-            page_count = int(
-                metadata["page_count"]
-            ) if metadata.get("page_count") else pdf_page_count(io.BytesIO(file_contents))
-            if notification["status"] == "validation-failed":
-                invalid_pages = metadata.get("invalid_pages")
-                invalid_pages = json.loads(invalid_pages) if invalid_pages else invalid_pages
-                error_message = get_letter_validation_error(
-                    metadata.get("message"), invalid_pages, page_count
-                )
-        except PdfReadError:
-            return render_template(
-                'views/notifications/invalid_precompiled_letter.html',
-                created_at=notification['created_at']
-            )
-    else:
-        page_count = get_page_count_for_letter(notification['template'], values=personalisation)
-        if page_count and page_count > LETTER_MAX_PAGE_COUNT:
-            # We check page count here to show the right error message for a letter that is too long.
-            # Another way to do this would be to get the status and error message from letter metadata.
-            # This would be a significant amount of work though, out of scope for this bug fix.
-            # This is because currently we do not pull the letter from S3 when showing preview.
-            # Instead, we generate letter preview based on the letter template and personalisation.
-            # Additionally, when a templated letter is sent via the api and the personalisation pushes the
-            # page count over 10 pages, it takes a while for validation status to come through.
-            # Checking page count here will enable us to show the error message even if the letter is not
-            # fully processed yet.
-            error_message = get_letter_validation_error(
-                "letter-too-long", [1], page_count
-            )
+    page_count = get_page_count_for_letter(notification['template'], values=personalisation)
```

Comment:
[question] why are we leaving this letter code in? It seemed like you're just deleting other letter stuff

Reply:
Nah, I'm good with a couple of smaller PRs